js/background.js

```diff
@@ -1407,87 +1408,101 @@ function isSafariContentBlockingAvailable() {
 // DEBUG INFO
 
 // Get debug info for bug reporting and ad reporting - returns an object
-function getDebugInfo() {
-
-    // An object, which contains info about AdBlock like
-    // subscribed filter lists, settings and other settings
-    var the_debug_info = {
-        filter_lists: [],
-        settings: [],
-        other_info: []
-    };
+// Callback: is defined for every browser except Safari
+function getDebugInfo(callback) {
 
-    // Process subscribed filter lists
-    var get_subscriptions = get_subscriptions_minus_text();
-    for (var id in get_subscriptions) {
-        if (get_subscriptions[id].subscribed) {
-            the_debug_info.filter_lists.push(id);
-            the_debug_info.filter_lists.push("  last updated: " + new Date(get_subscriptions[id].last_update).toUTCString());
-        }
+    // Is this installed build of CatBlock the official one?
+    function getBuildInfo(callback) {
+        chrome.management.getSelf(function(selfData) {
+            if (!selfData || !selfData.installType) {
+                return;
+            }
+
+            var installType = selfData.installType;
+
+            if (installType === "normal" || installType === "admin") {
+                callback("Stable");
+            } else if (installType === "development") {
+                callback("Developer");
+            } else {
+                callback("Unsupported");
+            }
```

Comment:
@kpeckett Please review changes in the latest commit (handling potential future copies).